src/Diagnostics.DataProviders/DataProviders/SupportObserverDataProvider.cs

```diff
@@ -72,20 +55,20 @@ public override async Task<dynamic> GetCertificatesInResourceGroupAsync(string s
         public override async Task<Dictionary<string, List<RuntimeSitenameTimeRange>>> GetRuntimeSiteSlotMap(string stampName, string siteName, string slotName = null)
         {
             if (string.IsNullOrWhiteSpace(stampName))
```

Comment:
You don't really need these conditional checks since `GetRuntimeSiteSlotMapInternal` is already doing it for you.